### ISSUE TITLE
Move data outside of the app logic

### ISSUE THREAD

**Issue by miyconst** [juliusedqvist/Python-game-1#4]
Currently all the text is hard coded into the game itself. Such data should be stored outside of the application.

The simplest way is to use `.json` files.

**Comment by juliusedqvist** [comment]
Good point, I actually haven't worked with json before nor did I know what it was during the development but I'll definitely look into that.
